example/lib/pie_chart/samples/pie_chart_sample2.dart

```diff
@@ -37,6 +37,7 @@ class PieChart2State extends State {
                           }
                         });
                       }),
+                      startDegreeOffset: -90,
```

Comment:
We don't need to update this variable, because we need to update the sample screenshot too, 
please revert this change.
Thank you!